packages/engine/src/networking/utils/getUser.ts

```diff
@@ -7,11 +7,21 @@ import { UserNameComponent } from '../../scene/components/UserNameComponent'
 import { Network } from '../classes/Network'
 import { NetworkObjectComponent } from '../components/NetworkObjectComponent'
 
-export function getUserEntityByName(name: string) {
+export function getUserEntityByName(name: string, localUserId) {
+  /*console.log('looking for entity: ' + name)
+  console.log('values: ' + JSON.stringify(Engine.defaultWorld.clients.values()))
   const client = Array.from(Engine.defaultWorld.clients.values()).find((c) => {
     return c.name === name
   })
-  return client ? useWorld().getUserAvatarEntity(client.userId) : undefined
+  return client ? useWorld().getUserAvatarEntity(client.userId) : undefined*/
+  const world = useWorld()
```

Comment:
If this function is meant to be called outside of a System, then it's preferable to use Engine.defaultWorld rather than `useWorld`

Reply:
updated it